pkg/metrics/metrics.go

```diff
@@ -0,0 +1,26 @@
+package metrics
+
+const (
+	// MetricComponentLabelKey is the key for the component of the metric.
+	MetricComponentLabelKey = "gpud_component"
+
+	// MetricLabelKey is the key for the label of the metric.
+	MetricLabelKey = "gpud_metric_label"
+)
+
+// Metric represents a metric row in the database table.
+type Metric struct {
+	// UnixSeconds represents the Unix timestamp of the metric.
+	UnixSeconds int64 `json:"unix_seconds"`
```

Comment:
We can record timestamps in seconds, but we declare it in milliseconds here to make it easier to change the time granularity in the future.